core/src/main/java/org/apache/iceberg/PositionDeletesTable.java

```diff
@@ -93,6 +95,16 @@ public Map<Integer, PartitionSpec> specs() {
     return specs;
   }
 
+  @Override
+  public Map<String, String> properties() {
+    // The write properties are needed by PositionDeletesRewriteAction,
+    // these properties should respect the ones of BaseTable.
+    return Collections.unmodifiableMap(
+        table().properties().entrySet().stream()
+            .filter(entry -> entry.getKey().startsWith("write."))
```

Comment:
I find  that all the write properties are needed for our PositionDeletesRewriteAction.  So I choose use to match the key prefix here instead of giving a fixed collection.